apps/src/templates/sectionAssessments/MultipleChoiceOverviewTable.jsx

```diff
@@ -25,12 +54,15 @@ class MultipleChoiceOverviewTable extends Component {
     questionAnswerData: PropTypes.arrayOf(questionAnswerDataPropType),
   };
 
-  state = {
-    [COLUMNS.NAME]: {
-      direction: 'desc',
-      position: 0
-    }
-  };
+  constructor(props) {
```

Comment:
what's this change do?

Reply:
Good question, the change is not necessary.  I did notice that before I made the change I used COLUMNS.NAME and didn't get an error and sorting worked as expected.  Maybe it is because  "transforms[sortable]" on the cell over writes it.  Thoughts..